.travis.yml

```diff
@@ -36,27 +28,20 @@ install:
   # Cpanfile upstream categorically refuses to implement the forcing of dependencies
   # (see https://github.com/miyagawa/cpanfile/issues/3) so we will have to keep this
   # here until either Net::FTPServer has been fixed or we stop using Test::FTP::Server
-  - cpanm -n IO::Scalar
-  - cpanm -v --installdeps --notest .
-  - cpanm -n Devel::Cover::Report::Coveralls
-  - cpanm -n DBD::SQLite
+  - cpanm --notest --sudo Archive::Zip Authen::PAM BSD::Resource File::Sync IO::Scalar
```

Comment:
This is super ugly: is there any sensible way to handle it? Tempted to replace Net::FTPServer